4.5_dry_up_with_inheritance/src/test/java/com/thinkful/drills/CharacterTest.java

```diff
@@ -0,0 +1,84 @@
+package com.thinkful.drills;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+import org.junit.Test;
+
+public class CharacterTest {
+
+  @Test public void testCreateNoArgs() {
+    Character character = new Character();
+    assertEquals("", character.getName());
+    assertEquals("", character.getDescription());
+    assertEquals(Character.INITIAL_HEALTH, character.getHealth());
+    assertEquals(Character.INITIAL_STRENGTH, character.getStrength(), 0.01);
+  }
+
+  @Test public void testCreateWithName() {
+    String name = "Gandalf";
+    Character character = new Character(name);
+    assertEquals(name, character.getName());
+    assertNotEquals("", character.getName());
+    assertEquals("", character.getDescription());
+    assertEquals(Character.INITIAL_HEALTH, character.getHealth());
+    assertEquals(Character.INITIAL_STRENGTH, character.getStrength(), 0.01);
+  }
+
+  @Test public void testCreateWithNameAndDescription() {
+    String name = "Gandalf";
+    String description = "Greyhelm";
+    Character character = new Character(name, description);
```

Comment:
the above mention changes to the constructor would make fewer tests needed and the overall code would be just as readable for a no arg constructor.
` 
    Character character = new Character();
    character.setName("Gandalf");
    character.setDescription("Greyhelm");
`